src/proto/FunctionRpc.proto

```diff
@@ -698,3 +701,24 @@ message ModelBindingData
 message CollectionModelBindingData {
   repeated ModelBindingData model_binding_data = 1;
 }
+
+// Retry policy which the worker sends the host when the worker indexes
+// a function. 
+message RetryPolicy {
```

Comment:
If we go with the explicit type, my guess is we want to match the types from the host instead of `string` for everything:
https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.Abstractions/Description/RetryOptions.cs